src/iface/interface.rs

```diff
@@ -393,6 +398,12 @@ let iface = builder.finalize(&mut device);
         self
     }
 
+    #[cfg(feature = "proto-sixlowpan-fragmentation")]
+    pub fn sixlowpan_fragments_cache_timeout(mut self, sec: u64) -> Self {
```

Comment:
`sec: u64` -> `duration: Duration`?

Reply:
Yeah, that is more ergonomic. 

@thvdveld do you think we should panic if the Duration is over 60s since that's in violation of the RFC?

Reply:
Probably log it, but I don't think that we should panic.